charts/capi-runtime-extensions/templates/deployment.yaml

```diff
@@ -29,7 +29,7 @@ spec:
         imagePullPolicy: "{{ .Values.image.pullPolicy }}"
         args:
         - --webhook-cert-dir=/runtimehooks-certs/
-        - --calicocni.defaultsNamespace={{ .Release.Namespace }}
+        - --calicocni.crs.defaultsNamespace={{ .Release.Namespace }}
```

Comment:
So in the future we could have something like this right?
```
- --calicocni.crs.defaultsNamespace={{ .Release.Namespace }}
- --calicocni.caaph.defaultsNamespace={{ .Release.Namespace }}
```

Reply:
Exactly that. Sound OK?

Reply:
Yep, was just confirming!